posthog/api/early_access_feature.py

```diff
@@ -270,9 +270,9 @@ def early_access_features(request: Request):
         )
 
     early_access_features = MinimalEarlyAccessFeatureSerializer(
-        EarlyAccessFeature.objects.filter(team_id=team.id, stage=EarlyAccessFeature.Stage.BETA).select_related(
-            "feature_flag"
-        ),
+        EarlyAccessFeature.objects.filter(
+            team__project_id=team.project_id, stage=EarlyAccessFeature.Stage.BETA
+        ).select_related("feature_flag"),
```

Comment:
seems fine to me but haven't touched these models in a minute, cc @havenbarnes for your feature management work since this may affect it.

Reply:
In this case I think early access features _should_ also be plainly project-level. It gets more complicated with feature flags, because you'll want its existence to be project-level, but rollout must be team-level (since persons are team-level)